integration-tests/general/common-test/src/main/kotlin/com/r3/conclave/integrationtests/general/commontest/TestUtils.kt

```diff
@@ -12,19 +12,10 @@ object TestUtils {
     }
 
     private fun getHardwareAttestationParams(): AttestationParameters {
-        return when {
-            // EPID vs DCAP can be detected because the drivers are different and have different names.
-            Path.of("/dev/sgx_enclave").exists() || Path.of("/dev/sgx/enclave").exists() -> {
-                AttestationParameters.DCAP()
-            }
-            Path.of("/dev/isgx").exists() -> {
-                val spid = OpaqueBytes.parse(System.getProperty("conclave.spid"))
-                val attestationKey = checkNotNull(System.getProperty("conclave.attestation-key"))
-                AttestationParameters.EPID(spid, attestationKey)
-            }
-            else -> throw UnsupportedOperationException(
-                "SGX does not appear to be available on this machine. Check kernel drivers."
-            )
-        }
+        return if (Path.of("/dev/sgx_enclave").exists() || Path.of("/dev/sgx/enclave").exists())
+            AttestationParameters.DCAP()
+        else -> throw UnsupportedOperationException(
+        "SGX does not appear to be available on this machine. Check kernel drivers."
```

Comment:
indent

Reply:
This is actually how IntelliJ indent this automatically. I feel that it does not see the scope after `else`. I suggest to use brackets to make this easier
```
    private fun getHardwareAttestationParams(): AttestationParameters {
        return if (Path.of("/dev/sgx_enclave").exists() || Path.of("/dev/sgx/enclave").exists()) {
            AttestationParameters.DCAP()
        } else -> {
            throw UnsupportedOperationException(
                "SGX does not appear to be available on this machine. Check kernel drivers."
            )
        }
    }
```